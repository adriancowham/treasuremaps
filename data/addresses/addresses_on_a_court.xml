<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339978699.html"> 
<title><![CDATA[ESTATE SALE (CARMICHAEL)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339978699.html</link> 
<description><![CDATA[Sale this weekend, April  22 and 23.  Sale will be from 9am to 4 pm.  1825 St. Ann Ct. Carmichael 95608 near Arden and Fair Oaks Blvd.
<br> 

<br> 
Great sale, don't miss it.
<br> 

<br> 
Very nice furniture  much of it antique.  Books, records,  costume jewelry, sewing, nice old linens, hundreds of buttons, marbles, toys, primitives, great old wooden boxes, c.1890 roll top desk, lots of picture frames, lots of beautiful dishes, bisque kewpies, japanese dolls,  like new Shop Smith, great work bench, tools, wonderful flower pots, patio tables, king size bed, piano, platform rocker, lots and lots of Christmas, both old and new.  Childrens books.  Side by side fridge, and a 1950's running  fridge.  Occupied Japan items, prints, picture frames.  Hundreds and hundreds of items.
<br> 

<br> 

<br> 
 
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=CARMICHAEL -->Location: CARMICHAEL
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:55:39-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339978699.html</dc:source> 
<dc:title><![CDATA[ESTATE SALE (CARMICHAEL)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:55:39-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339951038.html"> 
<title><![CDATA[Garage Sale: This Saturday, April 23, 9 a.m.-1 p.m. (Davis) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2339951038.html</link> 
<description><![CDATA[Garage sale Saturday, April 23, from 9:00 a.m. to 1:00 p.m.
<br> 
516 Hudson Ct. in Davis
<br> 
(No early birds, please)
<br> 

<br> 
Some good stuff for sale at even better prices!
<br> 

<br> 
•	Sturdy student desk
<br> 
•	Bookshelf (matches the student desk)
<br> 
•	Executive desk chair
<br> 
•	Fellows paper shredder (excellent condition)
<br> 
•	19-inch, 1999 Emerson television (works great)
<br> 
•	Television and media cabinet (has wheels)
<br> 
•	Digital converter box
<br> 
•	Phillips DVD player with remote
<br> 
•	Kitchen items (pots, pans, silverware, wine glasses…)
<br> 
•	Lava lamp
<br> 
•	Cream color bed duvet
<br> 
•	Walk-behind seed/fertilizer spreader
<br> 
•	Brand new bottle of antifreeze/coolant
<br> 
•	Big batch of baseball/football/basketball cards (approximately 6,000 in all)
<br> 
•	Two window air conditioners
<br> 
•	Classic boom box with CD player, dual cassette, and remote
<br> 
•	Small radio and cassette player
<br> 
•	Men’s clothing, including casual and dress shirts and pants (most sized medium to large, 32/34 waist, 34ish inseam)
<br> 
•	Women’s clothing, including casual and dress sweaters, shirts, pants, skirts, and dresses (sizes range from 6 to 9) and some shoes (size 8.5 and 9)
<br> 
•	Some baby/child items (infant swing, toddler bouncy toy, crib bumper…)
<br> 

<br> 
And there’s even more! Come have a look!
<br> 

<br> 
Driving Directions:
<br> 
From downtown Davis and from Highway 113, take Russell Blvd. west (staying in the left lane past Arlington) and turn right on Portage Bay Ave. E. Turn left on Hudson Ct.
<br> 

<br> 
Or just follow the signs.
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Davis -->Location: Davis
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T21:11:17-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339951038.html</dc:source> 
<dc:title><![CDATA[Garage Sale: This Saturday, April 23, 9 a.m.-1 p.m. (Davis) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T21:11:17-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339908535.html"> 
<title><![CDATA[Garage Sale -Saturday April 23 8AM-12PM (95829 - Elk Grove)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339908535.html</link> 
<description><![CDATA[Large family garage sale on Saturday April 23rd from 8AM-12PM.  East Sacramento/Elk grove area (off of Vintage Park Drive): 
<br> 
8071 Langdale Court
<br> 
Sacramento, CA 95829
<br> 

<br> 
Furniture
<br> 
kitchenware
<br> 
electronics
<br> 
clothes
<br> 
toys
<br> 
books
<br> 
movies
<br> 
booster seats
<br> 
LOTS of stuffed animals
<br> 
and more...
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=95829 - Elk Grove -->Location: 95829 - Elk Grove
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T20:28:52-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339908535.html</dc:source> 
<dc:title><![CDATA[Garage Sale -Saturday April 23 8AM-12PM (95829 - Elk Grove)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T20:28:52-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339802309.html"> 
<title><![CDATA[Roseville Garage Sale- Oven, Dishwasher, misc]]></title> 
<link>http://sacramento.craigslist.org/gms/2339802309.html</link> 
<description><![CDATA[Garage sale this Saturday (April 23rd) only at 104 Pennell Ct in Roseville.  
<br> 
The planets have finally alligned and caused my fiancee to be willing to get rid of some of her stuff!  Lots of candles, candle holders, lamps, purses, pillows, womens clothes and shoes.  Along with exercise equipment, heaters, and tons of books (I had to lose something in the deal).
<br> 
With these items, we are also selling a gas stove (black), dishwasher (black), entertainment center and nice dresser.  All going cheap.
<br> 
See you on Saturday!!!
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T19:03:29-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339802309.html</dc:source> 
<dc:title><![CDATA[Roseville Garage Sale- Oven, Dishwasher, misc]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T19:03:29-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339460982.html"> 
<title><![CDATA[2-Family Moving Sale (Citrus Heights)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339460982.html</link> 
<description><![CDATA[(Friday 4/22/2011 all day) and (saturday 4/23/2011from 7-11 am only)Saleing clothes , shoes, furniture and LOTS more stuff !!!!  Great deals Moving and can't take it with us :( my loss is your gain :) 7252 Lillivale ct , Citrus heights, 95621 off of tupelo ,antelope and hwy 80 look for the signs! or call wendy at 916-868-7353.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Citrus Heights -->Location: Citrus Heights
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T15:31:07-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339460982.html</dc:source> 
<dc:title><![CDATA[2-Family Moving Sale (Citrus Heights)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T15:31:07-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350516037.html"> 
<title><![CDATA[Multi-Family Sale (Elk Grove) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2350516037.html</link> 
<description><![CDATA[Multi family garage sale
<br> 
Saturday, April 30th, 8 am 
<br> 
No early shoppers
<br> 

<br> 
Clothes, Toys, small appliances
<br> 

<br> 
6105 Wild Fox Ct., Elk Grove, 95757<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Elk Grove -->Location: Elk Grove
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T23:10:29-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350516037.html</dc:source> 
<dc:title><![CDATA[Multi-Family Sale (Elk Grove) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T23:10:29-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350501965.html"> 
<title><![CDATA[MULTI FAMILY GARAGE AND MOVING SALE (6005 Sun Pearl Court Fair Oaks)]]></title> 
<link>http://sacramento.craigslist.org/gms/2350501965.html</link> 
<description><![CDATA[Lots of different items for sale. Some of the items include clothes, purses, jewelry, household, elvis items and many many more items too many to list. No reasonable offer refused. No personal checks and no early birds please. Sale is Saturday April 30th from 9:00 to 5:00. We are located near the corner of Fair Oaks and Greenback in the Sungarden Duplexes and Apartments. Look for the signs. <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=6005 Sun Pearl Court Fair Oaks -->Location: 6005 Sun Pearl Court Fair Oaks
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T22:43:19-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350501965.html</dc:source> 
<dc:title><![CDATA[MULTI FAMILY GARAGE AND MOVING SALE (6005 Sun Pearl Court Fair Oaks)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T22:43:19-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350500919.html"> 
<title><![CDATA[SATURDAY MORNING AMC GARAGE SALE (FOLSOM ) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2350500919.html</link> 
<description><![CDATA[SATURDAY ....APRIL 30TH......8AM UNTIL ALL IS GONE
<br> 
109 COFFEE BERRY CT.........FOLSOM- IN AMERICAN RIVER CANYON NEIGHBORHOOD
<br> 
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
<br> 

<br> 
BRAND NEW CLOSET ORGANIZER (UNOPENED BOX)
<br> 
BRAND NEW GARAGE SHELVING
<br> 
BRAND NEW TURKEY FRYER (STILL IN BOX)
<br> 
RUBBER MAID STORAGE CABINET
<br> 
GARAGE WALL STORAGE CABINET
<br> 
ELECTRIC SCOOTER
<br> 
COFFEE TABLE
<br> 
FOLDING TABLE
<br> 
LAWN FURNITURE
<br> 
ATARI GAME SET WITH A BUNCH OF GAMES
<br> 
BOARD GAMES
<br> 
TOOLS
<br> 
PICTURES
<br> 
VIDEOS
<br> 
FRAMES
<br> 
CLOTHES 
<br> 
MAKE UP
<br> 
COMFORTER AND PILLOWS
<br> 

<br> 
AND MUCH........MUCH MORE
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=FOLSOM  -->Location: FOLSOM 
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T22:41:30-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350500919.html</dc:source> 
<dc:title><![CDATA[SATURDAY MORNING AMC GARAGE SALE (FOLSOM ) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T22:41:30-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350397321.html"> 
<title><![CDATA[Davis multi-family yard sale (Sat, 4/30) (West Davis)]]></title> 
<link>http://sacramento.craigslist.org/gms/2350397321.html</link> 
<description><![CDATA[Multi-family garage sale; lots of stuff at amazingly low prices.
<br> 
Kids' bikes, bike trailer, toys, baby items, household items, clothing, and electronics.  
<br> 
Free stuff that's extremely good.
<br> 
Saturday, April 30th, 2011
<br> 
8:00 - 10:00 AM
<br> 
1745 Portola Court (near Shasta and Rio Grande)
<br> 
West Davis <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=West Davis -->Location: West Davis
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T20:33:00-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350397321.html</dc:source> 
<dc:title><![CDATA[Davis multi-family yard sale (Sat, 4/30) (West Davis)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T20:33:00-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2350074757.html"> 
<title><![CDATA[Garage Sale (Browns Valley) (Vacaville)]]></title> 
<link>http://sacramento.craigslist.org/gms/2350074757.html</link> 
<description><![CDATA[Garage Sale Friday, April 29th & Saturday April 30th 8am- ?? (no early birds please)
<br> 
Various household items, toys, baby clothes, furniture, and more
<br> 
948 Quate Court Vacaville (off Browns Valley Parkway)
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Vacaville -->Location: Vacaville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T16:42:39-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350074757.html</dc:source> 
<dc:title><![CDATA[Garage Sale (Browns Valley) (Vacaville)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T16:42:39-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2349930612.html"> 
<title><![CDATA[Huge Garage Sale (ROSEVILLE) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2349930612.html</link> 
<description><![CDATA[HOME DECOR, PICTURES, NINTENDO GAMEBOY,VASES, DESK TOP COMPUTER ( WORKS GREAT FROM MY BUSINESS THAT CLOSED) GARDEN, DESIGNER CLOTHES, COSTUME JEWELRY, VACUUM CLEANER,CHRISTMAS, IRON PIECES, DVD'S , CD'S ,FABRIC. SPACE RUGS, DECORATOR SHELVES, DISH  SETS.CANDLES , designer purses, wallets and shoesAND SO MUCH MORE..............
<br> 
EVERY ITEM IS CLEAN. WE ARE A SMOKE FREE AND ANIMAL  FREE HOME.
<br> 

<br> 
1104 Mill Estate Court
<br> 
Roseville, CA. 95661
<br> 
MAJOR CROSS STREETS CIRBY AND SUNRISE
<br> 
CIRBY RANCH DEVELOPMENT
<br> 
SAT. APRIL 30, 
<br> 
7:00-NOON  NO EARLY BIRDS!!!!   CASH ONLY  PLEASE<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=ROSEVILLE -->Location: ROSEVILLE
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T15:19:23-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2349930612.html</dc:source> 
<dc:title><![CDATA[Huge Garage Sale (ROSEVILLE) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T15:19:23-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2349862328.html"> 
<title><![CDATA[Huge Awesome Yard Sale (Rancho Cordova)]]></title> 
<link>http://sacramento.craigslist.org/gms/2349862328.html</link> 
<description><![CDATA[Well folks, my grandparents have finally decided to get rid of their old trusty and dusty RV that has served them well and was their mobile home all across this great state for the last 20+ years!
<br> 

<br> 
We'll be selling the RV (which is in GREAT working condition) to the highest bidder along with lots of other good stuff from inside! 
<br> 
Loot for purchasing will include but not limited to:
<br> 
clothing,
<br> 
jewelry,
<br> 
gently used appliances,
<br> 
household items,
<br> 
games,
<br> 
and other miscellaneous treasures! :)
<br> 
Come on down to our giant awesome yard sale THIS SUNDAY MORNING   May 1st, 2011  8AM - 1PM 
<br> 
10624 Chianti Court, Rancho Cordova
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Rancho Cordova -->Location: Rancho Cordova
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T14:43:45-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2349862328.html</dc:source> 
<dc:title><![CDATA[Huge Awesome Yard Sale (Rancho Cordova)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T14:43:45-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2349818217.html"> 
<title><![CDATA[Neighborhood Garage Sale (Rocklin)]]></title> 
<link>http://sacramento.craigslist.org/gms/2349818217.html</link> 
<description><![CDATA[This Saturday, April 30,  from 9 AM to 2 PM we will be participating in a neighborhood garage sale.  Items include furniture, television (55 inch), 2 Tivo DVRs, tools, rugs, garden equipment, and various household items.
<br> 

<br> 
4708 Gatwick Court
<br> 
Rocklin, CA 95677
<br> 

<br> 
Close to Sierra College Blvd an only 2 miles off of Hwy 80.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Rocklin -->Location: Rocklin
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T14:21:56-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2349818217.html</dc:source> 
<dc:title><![CDATA[Neighborhood Garage Sale (Rocklin)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T14:21:56-07:00</dcterms:issued> 
</item> 

</posts>