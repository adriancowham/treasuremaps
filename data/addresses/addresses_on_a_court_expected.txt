1825 St. Ann Ct
516 Hudson Ct
8071 Langdale Court
104 Pennell Ct
7252 Lillivale ct
6105 Wild Fox Ct
6005 Sun Pearl Court
109 COFFEE BERRY CT
1745 Portola Court
948 Quate Court
1104 Mill Estate Court
10624 Chianti Court
4708 Gatwick Court